flink-table/flink-table-planner-blink/src/test/scala/org/apache/flink/table/planner/plan/stream/sql/WindowRankTest.scala

```diff
@@ -398,4 +408,97 @@ class WindowRankTest extends TableTestBase {
     util.verifyRelPlan(sql)
   }
 
+  @Test
+  def testRankFunction(): Unit = {
+    val sql =
+      """
+        |SELECT window_start, window_end, window_time, a, cnt, sum_d, max_d, wAvg, uv
+        |FROM (
+        |SELECT *,
+        |   RANK() OVER(PARTITION BY window_start, window_end ORDER BY cnt DESC) as rownum
+        |FROM (
+        |  SELECT
+        |    a,
+        |    window_start,
+        |    window_end,
+        |    window_time,
+        |    count(*) as cnt,
+        |    sum(d) as sum_d,
+        |    max(d) filter (where b > 1000) as max_d,
+        |    weightedAvg(b, e) AS wAvg,
+        |    count(distinct c) AS uv
+        |  FROM TABLE(TUMBLE(TABLE MyTable, DESCRIPTOR(rowtime), INTERVAL '15' MINUTE))
+        |  GROUP BY a, window_start, window_end, window_time
+        |  )
+        |)
+        |WHERE rownum <= 3
+      """.stripMargin
+
+    thrown.expect(classOf[TableException])
+    thrown.expectMessage("RANK() on window rank is not supported currently.")
```

Comment:
What do you think about `RANK() function is not supported on Window TopN currently, only ROW_NUMBER() is supported.`